### ISSUE TITLE
[Enhancement] release Releases on Repository of Chocolatey packages

### ISSUE THREAD

**Issue by 4-FLOSS-Free-Libre-Open-Source-Software** [sirAndros/KeePassWinHello#22]
**Is your feature request related to a problem? Please describe.**
Hello, this is not an issue in itself. No. I'm always frustrated when to find & download and upgrade plgx and manually compare its integrity for security reasons.

**Describe the solution you'd like**
Just an suggestion. 
Would be nice to have a auto update feature for this primary security applications plugin from trusted source. The sha256 hash of PLRGX can be verified by installation automatically optionally if specified.  _For the lazy ones :) and_ to fix incompatibility issue with future of keepass version updates with plugin by getting all in one place for example.

I don't know if you have the spare time to maintain in parallel the "packages". Which basically would contains a script to URL of github [latest](https://github.com/JanisEst/KeePassQRCodeView/releases/latest) release asset. 

**Additional context**

_If you may want to take a look here if you don't know already:
[keepass plugins packages](https://chocolatey.org/packages?q=keepass&moderatorQueue=&prerelease=true&sortOrder=package-download-count)
(There are many plugins and packages count is growing. )_

**Comment by sirAndros** [comment]
Hello!
Finally, our package has been approved and now [you can check it out](https://chocolatey.org/packages/keepass-plugin-winhello) by installing latest version (2.2, the same that published [here](https://github.com/sirAndros/KeePassWinHello/releases/tag/2.2)).
All next releases will be published both on chocolatey.org and github.com.
